serializers/weather_serializer.js

```diff
@@ -0,0 +1,90 @@
+class WeatherSerializer {
+  static parse(weatherData) {
+    let results = {
+      today_forecast: this.current_forecast(weatherData.currently, weatherData),
+      daily_forecast: {
+        day_1: this.daily_forecast(weatherData.daily.data[1]),
+        day_2: this.daily_forecast(weatherData.daily.data[2]),
+        day_3: this.daily_forecast(weatherData.daily.data[3]),
+        day_4: this.daily_forecast(weatherData.daily.data[4]),
+        day_5: this.daily_forecast(weatherData.daily.data[5]),
+        day_6: this.daily_forecast(weatherData.daily.data[6]),
+        day_7: this.daily_forecast(weatherData.daily.data[7])
+      },
+      hourly_forecast: {
+        hour_1: this.hourly_forecast(weatherData.hourly.data[1]),
+        hour_2: this.hourly_forecast(weatherData.hourly.data[2]),
+        hour_3: this.hourly_forecast(weatherData.hourly.data[3]),
+        hour_4: this.hourly_forecast(weatherData.hourly.data[4]),
+        hour_5: this.hourly_forecast(weatherData.hourly.data[5]),
+        hour_6: this.hourly_forecast(weatherData.hourly.data[6]),
+        hour_7: this.hourly_forecast(weatherData.hourly.data[7]),
+        hour_8: this.hourly_forecast(weatherData.hourly.data[8])
+      }
+    };
+    return results
+  }
+
```

Comment:
Great job using a serializer and keeping your code SRP. For the hash outputs, it looks like it could be dry'd up if you had time later and wanted to circle back - but it looks like it's working for now!